src/commands/call.ts

```diff
@@ -57,6 +76,38 @@ export class ActorCallCommand extends ApifyCommand<typeof ActorCallCommand> {
             runOpts.memory = this.flags.memory;
         }
 
+        let input: Record<string, unknown> | undefined;
+
+        if (this.flags.input) {
+            switch (this.flags.input[0]) {
+                case '-': {
```

Comment:
In my opinion, this would fit better in the `input-file` option than here.

Reply:
Considering this is for stdin, I disagree 👀 why would it fit better in input-file?

Reply:
Well, my UNIX-y brain thinks of standard streams as of files :slightly_smiling_face:

Reply:
@netmilk thoughts? 🙏 
Basically, the way our CLI library works, is that you need to pass in `-` to a flag to make it read from stdin. I chose `--input` as the flag to accept stdin input since it made most sense (`wget xxx | apify call actor-here --input -`). We gotta pick if `input-file` makes more sense for stdin

Reply:
I meant for `--input` to ingest a string, for `--input-file` to ingest a file path.

I would love it if it could work without a flag at all:
```
jo foo=bar | apify call actor-here
```
or with `--input` flag without a value
```
jo foo=bar | apify call actor-here --input
```
**But** if those aren't an option, the `--input-file` is the way to go:
```
jo foo=bar | apify call actor-here --input-file=-
```
and/or
```
jo foo=bar | apify call actor-here --input-file -
```

I'm unsure if it's just a `bash` specific thing but I assume `-` is an alias for ` /dev/stdin` and that's a symlink to `/proc/self/fd/0`, thus "files". So I think of `-` as in it was a file.

Reply:
The `-` is a value that tells oclif to read from stdin (which is whatever process.stdin returns). I don't consider that a file at all (even if technically it is), so I personally would lean towards `--input` (since it's the same as doing `export VAR=""; apify call actor --input $VAR`, but I'll fold to whatever everyone agrees on. I could probably also try to implement it manually and read stdin, but not sure if that's going overkill

Reply:
I agree with what @netmilk wrote, but it's not a hill to die on for me.

Reply:
@vladfrangu, I'd appreciate it if we could follow up on or change the style guide. The easiest way of following the style guide is as follows:
- to **disable** `allowStdin` for `--input`
- to **enable** `allowStdin` for `--input-file`
- provide a convenience feature for `--input` without value to read from stdin, too, in the future (to create a separate issue for it)
- provide a convenience feature for the entire `actor call`, that piping to _stidin_ is an alias for `--input-file=-` (also in a separate issue)

Implementation-vise:
> since it's the same as doing `export VAR=""; apify call actor --input $VAR`

My poor understanding of Nodejs is that if you read from stdin, you are reading from an open, readable stream, so it is async – the same way you open a file for reading. Your example CLI is an atomic, sync one-off input.

The current "CLI Final Draft" is more of a "specification by example" rather than a formal spec. My takeaway here is that our specs, in general, might provide a more formal way of indicating the level of requirement (e.g., SHALL vs. MIUST) to make a clear distinction about how opinionated the spec is about what. ([RFC2119](https://datatracker.ietf.org/doc/html/rfc2119) for reference)